### PR TITLE
(v103) Bug 1779797 - Android: release branch string uplifts should auto-merge if CI is green

### PR DESCRIPTION
**Pull request by JohanLorenzo**
Ports https://github.com/mozilla-mobile/focus-android/pull/7397 and https://github.com/mozilla-mobile/focus-android/pull/7369 to the current release branch. No conflicts to solve.

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on `Show All Checks`,
2. click `Details` next to `build-focus-debug` or `build-klar-debug` for changes targeting Klar,
2. click `View task in Taskcluster`,
3. click the `Artifacts` row,
4. click to download any of the apks listed here which use an appropriate name for each CPU architecture.

### CODE REVIEW

File: taskcluster/ci/config.yml [hunk 1]
```diff
@@ -11,7 +11,15 @@ treeherder:
...
         'TL': 'Toolchain builds for Linux 64-bits'
 
-task-priority: highest
+task-priority:
+    by-project:
+        "focus-android": highest
+        "staging-focus-android": low
+        # This handles cases where a fork may end up not being named
+        # as one of the explicit names above. Ideally anything forked
+        # from "fenix" would inherit its setting, but "low" is the
```
Comment (ahal) [inline taskcluster/ci/config.yml:20]: nit: `s/fenix/focus-android`

**Comment by JohanLorenzo** [comment]
> I'm a bit confused, doesn't this implement a lot of the same stuff as #7397 ? Did you mean to submit both PRs?

You're right, it implements exactly the same stuff as #7397. This current PR is just a regular uplift to the current release branch `release_v103`.

**Comment by JohanLorenzo** [comment]
👋  @mozilla-mobile/focus-android-eng I have the same request as https://github.com/mozilla-mobile/focus-android/pull/7397#issuecomment-1191454632 🙂

**Comment by JohanLorenzo** [comment]
Oh, I seems I now have the rights to merge this PR, nice!

**Pull request merged**